02-oop-inheritance/socialClass.js

```diff
@@ -0,0 +1,17 @@
+"use strict";
+
+Object.defineProperty(exports, "__esModule", {
+    value: true
+});
+var Social = {
+
+    like: function like(friendName) {
+        console.log(friendName + " likes " + this.title);
+    },
+
+    share: function share(friendName) {
+        console.log("Share " + this.title + " with " + friendName);
+    }
+};
+
+exports.default = Social;
```

Comment:
You can just export the object.
Like: `exports.default = { // Social logic goes here`